batch7crud-roll4/src/main/webapp/resources/js/ajaxPost.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Created by pratishshr on 2/3/16.
+ */
+
+$('#ajaxForm').submit(function (e) {
+    e.preventDefault();
+    submitForm(this.action);
+});
+
+
+function submitForm(submitUrl) {
+    var firstName = $('#first-name').val();
+    var lastName = $('#last-name').val();
+    var station = $('#station').val();
+
+    var jsonObject = {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=javascript%3ASemicolon)